test/unit/uk/gov/hmrc/individualsmatchingapi/audit/AuditHelperSpec.scala

```diff
@@ -124,9 +95,11 @@ class AuditHelperSpec extends AsyncWordSpec with Matchers with MockitoSugar {
                                 |""".stripMargin)
 
       val capturedEvent = captor.getValue
-      capturedEvent.asInstanceOf[ExtendedDataEvent].auditSource shouldEqual "individuals-employments-api"
-      capturedEvent.asInstanceOf[ExtendedDataEvent].auditType shouldEqual "ApiResponseEvent"
-      capturedEvent.asInstanceOf[ExtendedDataEvent].detail shouldBe result
+      capturedEvent.asInstanceOf[ApiResponseEventModel].matchId shouldEqual matchId
```

Comment:
I'd prefer this capturedEvent.asInstanceOf[ApiResponseEventModel] by moved out into a class to remove the repitition.